grc/core/generator/flow_graph.py.mako

```diff
@@ -2,6 +2,7 @@
 <%
 from sys import version_info
 from gnuradio import eng_notation
+from __future__ import print_function
```

Comment:
doesn't the __future__ import need to come before any other imports?

Reply:
Yes, it should be at the top. Now corrected.